src/components/link-component/LinkComponent.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from 'react'
+
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LinkOutlinedIcon from '@mui/icons-material/LinkOutlined'
+
+import AppButton from '~/components/app-button/AppButton'
+
+import { ButtonVariantEnum, Link } from '~/types'
+
```

Comment:
remove space and one more between FC and box

Reply:
> remove space and one more between FC and box

fixed